framework/dotnet/BenchPress.TestFramework/MOCK_DELETE_LATER.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections.Generic;
+
+namespace BenchPress.TestEngine
+{
+  namespace Azure
+  {
+    public class Config
+    {
+      public void Set(Config config)
+      {
+      }
+    }
+
+    public class ResourceGroup
+    {
+      public object GetResourceGroup(string nameOrResourceId)
+      {
+        if (nameOrResourceId.Contains("/"))
+        {
+          return GetResourceGroupById(nameOrResourceId);
+        }
+        return GetResourceGroupByName(nameOrResourceId);
+      }
+
+      private object GetResourceGroupByName(string name)
+      {
+        return null;
+      }
+
+      private object GetResourceGroupById(string id)
+      {
+        return null;
+      }
+
+      public bool CheckResourceGroupExists(string nameOrResourceId)
+      {
+        return GetResourceGroup(nameOrResourceId) != null;
+      }
+    }
+
+    public class StorageAccount
+    {
+      public object GetStorageAccount(string nameOrResourceId)
+      {
+        if (nameOrResourceId.Contains("/"))
```

Comment:
Mocks shouldn't have decision logic going on inside them

Reply:
it's also a placeholder reminder for us to implement similar logic in test engine core

Reply:
The only reminder we should need is a failing test! That is the reason we are going with a test first approach, if the test does not fail without this line, then it is not a valid test in the first place.